docs/csharp/language-reference/operators/remainder-operator.md

```diff
@@ -8,21 +8,25 @@ ms.topic: "article"
 f1_keywords: 
   - "%_CSharpKeyword"
 helpviewer_keywords: 
-  - "modulus operator [C#]"
+  - "remainder operator [C#]"
   - "% operator [C#]"
 ms.assetid: 3b74f4f9-fd9c-45e7-84fa-c8d71a0dfad7
 caps.latest.revision: 15
 author: "BillWagner"
 ms.author: "wiwagn"
 ---
 # % Operator (C# Reference)
-The `%` operator computes the remainder after dividing its first operand by its second. All numeric types have predefined remainder operators.  
+The remainder operator (`%`) computes the remainder after dividing its first operand by its second. All numeric types have predefined remainder operators. 
   
 ## Remarks  
+ The formula `a % b` will always return a value on the range `(-b, b)`, exclusive (it can never return `b` or `-b`).
+  
+ This is not to be confused with canonical modulus, which satisfies a similar rule but with floored division and returns values on the range `[0, b)`. C# does not have an operator for canonical modulus. However, the behavior is the same for positive dividends.
```

Comment:
Not sure, if this paragraph is necessary. Let's make the definition above is clear enough not to be confusing.

Reply:
I would keep this paragraph in the remarks. It is clarifying for any reader that is trying to make the distinction between remainder and modulus.